index.html

```diff
@@ -30,7 +30,7 @@ <h1>Get ready to Spot the Surveillance...</h1>
         <img id="congrats-card" src="assets/images/congrats-card-replay.png">
         <img id="drone-card"  src="assets/images/drone-card.png">
         <img id="intro-card" src="assets/images/intro-card.png">
-        <img id="eff-logo" src="assets/images/eff-logo-black.svg">
+        <img id="eff-logo" src="assets/images/eff-logo-black.png">
```

Comment:
can you compress the PNG with `pngquant`? not a biggie, but was able to shave off 2 KB.